credentials/src/test/java/com/thousandeyes/sdk/credentials/CredentialsApiTest.java

```diff
@@ -87,9 +112,23 @@ public void createCredentialRequestAndResponseDeserializationTest()
                   "id" : "3247"
                 }
                                   """;
+        int statusCode = 201;
+        String contentType = "application/json";
         CredentialWithoutValue mappedResponse = 
                 mapper.readValue(responseBodyJson, CredentialWithoutValue.class);
         assertNotNull(mappedResponse);
+
+        String path = "/credentials";
+        stubFor(post(urlPathTemplate(path))
+                        .withRequestBody(equalToJson(requestBodyJson))
+                        .willReturn(aResponse()
+                                            .withHeader(AUTHORIZATION, BEARER_TOKEN)
+                                            .withHeader(CONTENT_TYPE, contentType)
```

Comment:
shouldn't this 2 be on the request? 🤔

Reply:
you're absolutely right, will change that on the generator

Reply:
cant really put accept cause we customize it during generating, but added the request content type:
https://github.com/thousandeyes/thousandeyes-sdk-generator/pull/39/commits/5d59ca23676d175f6080a3337dacce132c8c2989

Reply:
Ive pushed an update to credentials test only so we can see how it looks